### ISSUE TITLE
Describing the JSON structure.

### ISSUE THREAD

**Issue by spydmobile** [CWFMF/wx_specification#3]
So my first thought is we should be using JSON Schema officially to describe the JOSN structure.

for those not familiar:
https://json-schema.org/learn/getting-started-step-by-step

The rationale is that it will be far easier to implement these entities/structures if they are described in JSON schema vs plain JSON. 

The work put into the scheme can then be later used programmatically to generate entities etc.

@jordan-evens, you may be doing this already, but if not I am asking if we should do it moving forward.

**Comment by jordan-evens** [comment]
Yeah we definitely need formal specs for things and not just verbiage. I have the first stab at the schemas in the schemas subfolder, and the example folders were some basic code to play around with them to see if they make sense. The readme for the repo is basically the thought process and considerations that went into making the files in the schema folder.
